src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -308,13 +311,19 @@ public async Task InitializeAsync(IEnumerable<FunctionMetadata> functions, Cance
             }
             else
             {
-                _processStartupInterval = workerConfig.CountOptions.ProcessStartupInterval;
-                _restartWait = workerConfig.CountOptions.ProcessRestartInterval;
-                _shutdownTimeout = workerConfig.CountOptions.ProcessShutdownTimeout;
+                _processStartupInterval = workerConfig?.CountOptions?.ProcessStartupInterval ?? _defaultProcessStartupInterval;
```

Comment:
yes, since there will not be a `workerConfig`, this code throws a NRE. (we did not get to this state in 4.35 because we threw `new HostInitializationException($"Did not find functions with language [{workerRuntime}].");`)